.babelrc

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Is causing issues with testing. While this works with VueJS CLI buildouts, there is a problem with the Nuxt stack in using this configuration.